DownFall/TargetingViewModel.swift

```diff
@@ -125,6 +164,22 @@ class TargetingViewModel: Targeting {
                     self?.tiles = endTiles
                 }
                 
+                if let inputType = trans.inputType,
+                    case InputType.itemUsed(_) = inputType,
+                    let tiles = trans.endTiles,
+                    let playerCoord = getTilePosition(.player(.zero), tiles: tiles),
+                    case TileType.player(let data) = tiles[playerCoord].type
+                {
+                    self?.inventory = data.abilities
+                }
+                
+            case .boardBuilt:
+                guard let self = self,
+                    let tiles = input.endTilesStruct else { return }
+                
+                if let playerData = playerData(in: tiles) {
+                    self.inventory = playerData.abilities
+                }
```

Comment:
Is there any other data that we can put into the ViewModel?

Reply:
not much.  things to consider would be the item detail view.  Because that is a huge huge function.  scrolling.  button handling. but nothing that seems to belong in the viewmodel